src/rshim.c

```diff
@@ -2359,8 +2464,12 @@ int rshim_register(rshim_backend_t *bd)
   }
 
   rc = rshim_access_check(bd);
-  if (rc)
+  if (rc) {
+    if (rc == -EEXIST) {
+      RSHIM_INFO("another backend already attached\n");
```

Comment:
why did you move this message here? I see that rshim_access_check is called in rshim_fuse.c as well

Reply:
Good question. In the previous design, this message was printed in `rshim_access_check()`, which could cause log-overwhelming issues when we have to call it (and fail) many times. That's exactly what my previous monitoring code does. It had to call `rshim_access_check()` every 300ms when in that "zombie" mode (aka, NIC state changed from locked to unlocked, but rshim driver unable to be attach due to BMC already attached). Now the new code will enter drop mode directly, so technically this change is not needed any more. While it's still a good idea to decouple the logic and message print, I will reverse this change so we only do minimum to achieve the desired fix.